src/components/utilities/parseMclSettings.js

```diff
@@ -0,0 +1,6 @@
+import { defaultSettingsValues } from '../../constants';
+
+export default function parseMclSettings(settings, name, mcl) {
```

Comment:
Just thinking this through, I'd say lets change this function to take `parseMclsPageSize(settings, mcl)` as we are only dealing with `pageSize` now. I dont see us extending this function anytime soon and keep it this way will only lead to further confusion. What do you think?